src/pages/search/components/Results.jsx

```diff
@@ -34,27 +40,6 @@ function mapDispatchToProps(dispatch) {
   }, dispatch);
 }
 
-const faNames = {
-  challenge: 'free-code-camp',
-  guides: 'file-text',
-  youtube: 'youtube-play'
-};
-
-function FAIcon({ name }) {
```

Comment:
Good call on refactoring this 👍

Reply:
I don't know why I put a wrapper around it in the first place 😕